main.go

```diff
@@ -387,7 +401,7 @@ func runWatcher(fileChangeCommand string, dirs []string, killServer bool) error
 	errorMessage := ""
 	var err error
 
-	ControllerDebug.log("Starting watcher for ON_CHANGE command: " + fileChangeCommand)
+	ControllerDebug.log("Starting watcher: " + fileChangeCommand)
```

Comment:
I meant remove the command from the log message entirely and just say "Starting watcher". Is that okay with you? You log the command later right?

Reply:
I fixed it.  See if it is OK now.